README.md

```diff
@@ -1,2 +1,25 @@
 # Collabo Web App
-Content coming soon
+
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
```

Comment:
**Review comment 3:** We have not set up @all-contributors bot for this repo, so you can remove the all contributors badge and comments for now. This part is addressed in another issue ticket.